src/main/java/nl/tudelft/pixelperfect/event/Event.java

```diff
@@ -104,6 +112,16 @@ public boolean isExpired(long time) {
   }
 
   /**
+   * Updates the time left variable for the event.
+   * 
+   * @param currentTime
+   *          the current time of the game
+   */
+  public void updateTimeLeft(long currentTime) {
+    this.timeLeft = this.timestamp - currentTime;
+  }
```

Comment:
Why seperate updateTimeLeft and getTimeLeft, and not just do this simple calculation in the getter